test/e2e/quickstart/test.sh

```diff
@@ -20,4 +24,36 @@ EOF
 curl http://localhost:8000/openai/v1/completions \
   --max-time 900 \
   -H "Content-Type: application/json" \
-  -d '{"model": "gemma2-2b-cpu", "prompt": "Who was the first president of the United States?", "max_tokens": 40}'
+  -d '{"model": "deepseek-r1-1.5b-cpu", "prompt": "Who was the first president of the United States?", "max_tokens": 40}'
+
+
+DEEPSEEK_POD=$(kubectl get pod -l model=deepseek-r1-1.5b-cpu -o jsonpath='{.items[0].metadata.name}')
+
+# Test to ensure that model url can be updated without requests failing
+kubectl patch model deepseek-r1-1.5b-cpu --type=merge -p '{"spec": {"url": "ollama://qwen2.5:0.5b"}}'
+
+# Set maximum number of retries
+MAX_RETRIES=120
+retry_count=0
+
+# Continiously run curl requests to the model until the new pod is ready
+while true; do
```

Comment:
Prefer to use retry func:

https://github.com/substratusai/kubeai/blob/bc6c60fd50535c2802a1b274dbd175e5d637b355/test/e2e/common.sh#L3

Reply:
ps: it is already included since this script is imported